src/pages/Facility/services/pharmacy/hooks/useMedicationBill.ts

```
@@ -437,10 +445,98 @@ export function useMedicationBill({
+          quantity: sel.quantity,
+        })),
+      );
+
+      appliedIndices.push(index);
+    });
+
+    // Remove applied indices from pending set
+    if (appliedIndices.length > 0) {
+      setPendingSubstitutionAutoSelections((prev) => {
+        const next = new Set(prev);
+        appliedIndices.forEach((i) => next.delete(i));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)